src/components/entity/filter/EntityFilterFacetPanels.vue

```diff
@@ -0,0 +1,203 @@
+                    :value="getEntityFilters['Date_max']"
+                    label="To:"
+                    :append-icon="mdiCalendar"
+                    readonly
+                    clearable
+                    @click:clear="resetMaxDate"
+                    v-bind="attrs"
+                    v-on="on"
+                  ></v-text-field>
+                </div>
+              </template>
+              <v-date-picker
+                @input="menuTo = false"
+                @change="handleMaxDateChange"
+                header-color="#38598A"
+              ></v-date-picker>
+            </v-menu>
+          </v-col>
+          <v-spacer></v-spacer>
+        </v-row>
+      </template>
+    </CustomFilterFacetPanel>
+
+    <CustomFilterFacetPanel>
+      <template #title>Show Expired</template>
+      <template #content>
+        <v-container>
+          <v-col>
+            <p>
+              Show Expired
+              <v-switch
+                :input-value="
+                  isEntityFilterActive('Show_expired', getEntityFilters)
+                "
+                :value="isEntityFilterActive('Show_expired', getEntityFilters)"
+                @change="handleSwitchChange"
+              ></v-switch>
+            </p>
+          </v-col>
+        </v-container>
+      </template>
+    </CustomFilterFacetPanel>
+  </v-expansion-panels>
+</template>
+
+<script lang="ts">
+import { IEntityFacetField } from "@/interfaces/api/v2/entityFilter.interface";
+import { Component, Vue } from "vue-property-decorator";
+import { mapActions, mapGetters } from "vuex";
+import EntityFilterFacetPanel from "@/components/entity/filter/EntityFilterFacetPanel.vue";
+import CustomFilterFacetPanel from "@/components/entity/filter/CustomFilterFacetPanel.vue";
+import { Filter } from "@/store/modules/entity";
+import { isEntityFilterActive } from "@/utils/entity";
+
+interface Data {
+  menuFrom: boolean;
+  menuTo: boolean;
+  fromDate: string;
+  toDate: string;
+  panel: number[];
+}
+
+@Component({
+  components: {
+    EntityFilterFacetPanel,
+    CustomFilterFacetPanel,
+  },
+  computed: {
+    ...mapGetters([
+      "searchTopicFacets",
+      "getAuthorities",
+      "getCredentialTypes",
+      "getRegistrationTypes",
+      "getEntityFilters",
+      "mdiCalendar",
+    ]),
+  },
+  methods: {
+    ...mapActions(["setFilter"]),
+  },
+})
+export default class EntityFilterFacetPanels extends Vue {
+  private getAuthorities!: Array<IEntityFacetField>;
+  private getCredentialTypes!: Array<IEntityFacetField>;
+  private getRegistrationTypes!: Array<IEntityFacetField>;
+  getEntityFilters!: Filter;
+  setFilter!: (filter: Filter) => void;
+  isEntityFilterActive: (
+    filterField: string,
+    getEntityFilters: Filter,
+    filterString?: string
+  ) => boolean = isEntityFilterActive;
+  data(): Data {
+    return {
+      menuFrom: false,
+      menuTo: false,
+      fromDate: "",
+      toDate: "",
+      panel: [0, 1, 4],
```

Comment:
Is it ok to have these values hardcoded?

Reply:
I think so, because these are the default panels that will be open in the entity filters when you navigate to the page